src/cps/pc_compat/pc.y

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Is there an existing BNF grammar for pkg config files we can refer to or quote?

Reply:
Not that I know of. But there probably is? I was using https://manpages.ubuntu.com/manpages/jammy/man5/pc.5.html as my reference

Reply:
pkgconf also uses a hand rolled recursive descent parser, but that man page is from pkgconf. My preference would be to emulate pkgconf rather than pkg-config if possible, since it actually has developers, and most distros are shipping pkgconf with a symlink to pkg-config

Reply:
I guess I'm fine with landing in the current form, and if we encounter compatibility issues then we can make tweaks as needed

Reply:
I have a branch of pkgconf (that probably needs some additional work) that ports the kyua tests to a python runner. I can try to get those around for cps-config

Reply:
Yup. When you do, feel free to ping me to fix any issues (I bet there would be a lot)